usersync/usersync.go

```diff
@@ -30,6 +30,7 @@ func NewSyncerMap(cfg *config.Configuration) map[openrtb_ext.BidderName]Usersync
 		openrtb_ext.BidderPubmatic:   NewPubmaticSyncer(cfg.ExternalURL),
 		openrtb_ext.BidderPulsepoint: NewPulsepointSyncer(cfg.ExternalURL),
 		openrtb_ext.BidderRubicon:    NewRubiconSyncer(cfg.Adapters["rubicon"].UserSyncURL),
+		openrtb_ext.BidderAdform:     NewAdformSyncer(cfg.Adapters["adform"].UserSyncURL, cfg.ExternalURL),
```

Comment:
Going to merge this PR because it's got the approvals, but... one open issue you should be aware of.

The way this is written, a PBS host company won't know what to set their config value to be in order to make usersyncs work with your bidder... so you'll be at a disadvantage compared to the other bidders in the project.

To level the playing field, you may want to add a `viper.SetDefault()` line to `init()` in `pbs_light.go`. Or, if there's no reason why one PBS deploy would use a different sync URL than any other, just eliminate the config altogether and hardcode the URL as a `const` in your Usersyncer.

This can be addressed in a separate PR, though, and is not required for the project.